tests/posit/connect/oauth/test_integrations.py

```diff
@@ -1,61 +1,11 @@
-from unittest import mock
-
 import responses
 from responses import matchers
 
 from posit.connect.client import Client
-from posit.connect.oauth.associations import IntegrationAssociations
-from posit.connect.oauth.integrations import Integration
 
 from ..api import load_mock  # type: ignore
 
 
-class TestIntegrationAttributes:
-    @classmethod
-    def setup_class(cls):
-        guid = "22644575-a27b-4118-ad06-e24459b05126"
-        fake_item = load_mock(f"v1/oauth/integrations/{guid}.json")
```

Comment:
With both tests being removed, should we remove the mock files?

Reply:
The mocks are also used in other tests below.